understanding/21/non-text-contrast.html

```diff
@@ -31,14 +31,14 @@ <h3>Active User Interface Components</h3>
 
 				<figure>
 					<img alt="Two buttons, the first with no visual indicator except text saying 'button'. The second is the same but with an added grey border." src="img/minimal-button.png" /> 
-					<figcaption>An active control without a visual boundary, but with a focus indicator when focused.</figcaption>
+					<figcaption>An active control without a visual boundary, but with a focus indicator when focused</figcaption>
 				</figure>
 
-				<p>The <a href="use-of-color">Use of Color</a> success criteria addresses changing <strong>only the color</strong> of an object (or text) without otherwise altering the object's form. If a controls indicator of state (e.g. button background) varies only by color this is acceptable if the luminosity contrast ratio between the colors of the states differ by at least 3:1, or if there is another indicator of state. For example, a text link that only differs from adjacent text using color and there is no other visual indication that the text is linked (e.g. the link underline is removed) needs to ensure that the link color meets the 3:1 luminosity contrast ratio relative to the non-linked text color in order to meet the related Success Criteria 1.4.1.</p>
+				<p>The <a href="use-of-color">Use of Color</a> success criteria addresses changing <strong>only the color</strong> of an object (or text) without otherwise altering the object's form. If a control's indicator of state (e.g., button background) varies only by color this is acceptable if the luminosity contrast ratio between the colors of the states differ by at least 3:1, or if there is another indicator of state. For example, a text link that only differs from adjacent text using color where there is no other visual indication that the text is linked (i.e., the link underline is removed) needs to ensure that the link color meets the 3:1 luminosity contrast ratio relative to the non-linked text color in order to meet the related Success Criteria 1.4.1.</p>
```

Comment:
@mbgower this is the only one that I'm not 100% on. This feels like an example but maybe needs to be better clarified as an example?  How about:

(e.g., black paragraph text with blue links with the link underline removed)

Reply:
I assume you're talking about where I change the e.g. to i.e., ? (The only other change here is adding in a comma after the prior e.g.).
If you think the actual wording needs to be changed, that can be done, but I simply did a search for e.g., style, not for content. I changed this to i.e. (_id est_, 'in other words') because the _only_ way I could think of have no other indication of text being a link is to remove the underline. So my feeling was that this isn't an example, it's a rewording. Hence the i.e.

BTW, since we're talking about wording, I find this whole thing a little awkwardly phrased. I believe this would both make more sense AND be more accurate

> For example, if a text link only differs from adjacent text using color (because the browser-supplied persistent underline for the link has been removed by the author) the author needs to ensure that the link color meets the 3:1 luminosity contrast ratio relative to the non-linked text color in order to meet the related Success Criteria 1.4.1.

I also want to add in an additional note saying the underline has to appear on focus, but maybe that is just making the use case too messy when we're only trying to address...

You know what?! I think this link example is really not a good example here. "If in doubt, take it out". I would be inclined to strip the entire final sentence out and just end it at "...another indicator of state." Bringing in a link example is just making it too messy!

Reply:
It is, after all, supposed to be about NON-TEXT contrast.  Geez, one pulls a thread and the whole sweater begins to unravel!

Reply:
I would be happy to take the parenthetical out of the sentence. The sentence is already a "for example" so it should be simple enough without an example.

Reply:
The link example is good, in the "for example" - the contrast is about the non-text aspects of the link.

Reply:
@awkawk, 

Here's what I think we should do. Let's restrict the changes to the ones I made, which is almost completely concerned with swapping out e.g. for e.g.,  (and replacing one e.g. with an i.e.,)

I will then actually tackle a larger review of this Understanding doc in a separate issue https://github.com/w3c/wcag/issues/492. 

-- 
In relation to where this conversation has gone, I'd much rather have the example be about non-text. For instance, it could talk about inversing the foreground and background colors (as is the case with the 3rd blue-on-white button, where it becomes white on blue on focus) or thickening the border (as is the case with the first blue-on-white button that gets a wider blue border on focus).  And as I write that, I'm wondering if the figures should be numbered?

My concern with the example as it stands is that if there is no underline indicating link, then the color example is wholly about text, not non-text.  But, I suggest we just ignore that for now and get these 'easy to pass' editorial changes in so that  481 can get closed.

Reply:
ok